modules/sbo-connecting-spring-petclinic-sample-app-to-postgresql-database-service.adoc

```diff
@@ -42,81 +37,50 @@ The output verifies that the `ServiceBinding` CR is created to project the bindi
 +
 .Example output
 ----
-servicebinding.binding.operators.coreos.com/spring-petclinic-rest created
+servicebinding.binding.operators.coreos.com/spring-petclinic created
 ----
 
-. To verify that the binding is successful, check the status conditions of the binding resource:
+. Verify that the request for service binding is successful:
```

Comment:
How does a user know that the request was successful? What should the servicebindings status be?